man/Makefile.am

```diff
@@ -1,5 +1,11 @@
 AUTOMAKE_OPTIONS = gnu
 
```

Comment:
I thought sphinx-build was able to create a single man page from a single rst file. Is it not the case ?